newsApi.js

```diff
@@ -0,0 +1,24 @@
+const apiKey = require('./apiKey')
+
+class NewsApi {
+
+  loadHeadlines(callback) {
+    fetch(`https://content.guardianapis.com/search?api-key=${apiKey}&type=article&show-fields=thumbnail&show-fields=all`)
+      .then(response => response.json())
+      .then(data => {
+        console.log(data.response.results),
+        callback(data.response.results)})
+      .catch((error) => console.log(error))
+  }
+
+  searchHeadlines(searchTerm = '', callback) {
+    fetch(`https://content.guardianapis.com/search?q=${searchTerm}&query-fields=headline&show-fields=thumbnail,headline,byline&order-by=newest&api-key=${apiKey}`)
+      .then(response => response.json())
+      .then(data => {
+        console.log(data.response.results),
+        callback(data.response.results)})
+      .catch((error) => console.log(error))
+  }
+}
+
```

Comment:
I very much like that you have accessed response.results in here and passed that data forward, seems very neat! And good practice with using catch too!